### ISSUE TITLE
Add RBAC support

### ISSUE THREAD

**Issue by radu-matei** [radu-matei/kube-toolkit#10]

**Comment by mumoshu** [comment]
What do you want to cover for this?
If you're referring to a serviceaccount and associated RBAC permissions used by the server component of a kube-toolkit-generated app, can we do it in #25?

For reference, I suppose we could do it in a helm chart like e.g. Brigade do in: https://github.com/Azure/brigade/blob/master/charts/brigade/templates/api-role.yaml

**Comment by radu-matei** [comment]
Yes, right now RBAC permissions of the server component is what I would like to have.

What I like at the moment is that you don't have any prerequisites for installing a tool built with `kube-toolkit`.

But yeah, having RBAC might justify deploying this with Helm.
Thoughts?
